app/controllers/application_controller.rb

```diff
@@ -22,12 +23,11 @@ def append_to_log(params)
   # Rails' instrumentation code, and is run after each request.
   def append_info_to_payload(payload)
     super
-    if @custom_log_items
-      payload[:custom_log_items] = @custom_log_items
-    end
-  end
 
-  # :nocov:
+    append_to_log(request_id: RequestStore.store[:request_id])
```

Comment:
What would you change the name of the action to? `store_and_log_request_id` ?

I don't even like having the `store_request_id` callback thinking more about it, I think it would make most sense as a middleware, which would mean that it could be tested more easily. What do you think?